libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/AttachmentStreamConnector.java

```diff
@@ -106,13 +107,25 @@ public InputStream unsafeGetAttachmentStream(AttachmentContent attachment) throw
         PipedOutputStream out = new PipedOutputStream(in);
 
         new Thread(() -> {
+            HashMap<String, Integer> addedFileNames = new HashMap<>();
```

Comment:
minor comment: the variable name does not describe its use good enought.

Reply:
changed variable name to `fileNameUsageMap`.